packages/logs/src/domain/logsCollection/console/consoleCollection.spec.ts

```diff
@@ -1,120 +1,100 @@
-      createSender(sendLogSpy)
+      lifeCycle
     ))
 
     /* eslint-disable-next-line no-console */
     console.log('foo', 'bar')
 
-    expect(sendLogSpy).toHaveBeenCalledWith({
+    expect(rawLogs[0].rawLog).toEqual({
       message: 'foo bar',
       status: StatusType.info,
       origin: ErrorSource.CONSOLE,
+      error: undefined,
     })
 
     expect(consoleLogSpy).toHaveBeenCalled()
   })
 
   it('should not send console logs when ff forward-logs is disabled', () => {
-    ;({ stop: stopConsolCollection } = startConsoleCollection(
+    ;({ stop: stopConsoleCollection } = startConsoleCollection(
       validateAndBuildLogsConfiguration({ ...initConfiguration, forwardConsoleLogs: ['log'] })!,
-      createSender(sendLogSpy)
+      lifeCycle
     ))
 
     /* eslint-disable-next-line no-console */
     console.log('foo', 'bar')
 
-    expect(sendLogSpy).not.toHaveBeenCalled()
+    expect(rawLogs.length).toEqual(0)
     expect(consoleLogSpy).toHaveBeenCalled()
   })
 
   it('should send console errors with "console" origin when ff forward-logs is enabled', () => {
     updateExperimentalFeatures(['forward-logs'])
-    ;({ stop: stopConsolCollection } = startConsoleCollection(
+    ;({ stop: stopConsoleCollection } = startConsoleCollection(
       validateAndBuildLogsConfiguration({ ...initConfiguration, forwardErrorsToLogs: true })!,
-      createSender(sendLogSpy)
+      lifeCycle
     ))
 
     /* eslint-disable-next-line no-console */
     console.error('foo', 'bar')
-
-    expect(sendLogSpy.calls.mostRecent().args[0].origin).toEqual(ErrorSource.CONSOLE)
+    expect(rawLogs[0].rawLog.origin).toEqual(ErrorSource.CONSOLE)
   })
 
   it('should not send console errors with "console" origin when ff forward-logs is disabled', () => {
-    ;({ stop: stopConsolCollection } = startConsoleCollection(
+    ;({ stop: stopConsoleCollection } = startConsoleCollection(
       validateAndBuildLogsConfiguration({ ...initConfiguration, forwardErrorsToLogs: true })!,
-      createSender(sendLogSpy)
+      lifeCycle
     ))
 
     /* eslint-disable-next-line no-console */
     console.error('foo', 'bar')
 
-    expect(sendLogSpy.calls.mostRecent().args[0].origin).not.toBeDefined()
+    expect(rawLogs[0].rawLog.origin).not.toBeDefined()
   })
 
   it('console error should have an error object defined', () => {
-    ;({ stop: stopConsolCollection } = startConsoleCollection(
+    ;({ stop: stopConsoleCollection } = startConsoleCollection(
       validateAndBuildLogsConfiguration({ ...initConfiguration, forwardErrorsToLogs: true })!,
-      createSender(sendLogSpy)
+      lifeCycle
     ))
 
     /* eslint-disable-next-line no-console */
     console.error('foo', 'bar')
 
-    expect(sendLogSpy.calls.mostRecent().args[0].error).toEqual({
+    expect(rawLogs[0].rawLog.error).toEqual({
       origin: ErrorSource.CONSOLE,
       stack: undefined,
     })
   })
-
-  it('should not print the log twice when console handler is enabled', () => {
```

Comment:
This test now requires that `consoleCollection` and `loggerCollection` start so I moved to startLogs.spec